homework-2/src/scss/core/_cybersecurity.scss

```diff
@@ -0,0 +1,163 @@
+.cybersecurity {
+    min-width: 375px;
+    height: 100%;
+    padding: 0 20px;
+    &__section {
+        display: flex;
+        flex-direction: column;
+        min-width: 335px;
+        width: 100%;
+        height: 100%;
+    }
+    &__page-title {
+        font-size: 32px;
+        font-family: $font-text;
+        margin: 0 0 24px 0;
+    }
+    &__left {
+        display: flex;
+        flex-direction: column;
+        min-width: 335px;
+        width: 100%;
+        height: 100%;
+    }
+    &__right {
+        display: flex;
+        flex-direction: column;
+        min-width: 335px;
+        width: 100%;
+        height: 100%;
+    }
+    &__area {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        min-width: 335px;
+        width: 100%;
+        height: 170px;
+        padding: 20px;
+        border-left: 1.5px solid $color-button-hr;
+        box-shadow: 0px 12px 24px rgba(127, 127, 127, 0.3);
+        border-radius: 5px;
+        margin: 0 0 25px 0;
+    }
+    &__area-text {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        padding-right: 20px;
+    }
+    &__sub-tittle {
+        font-size: 24px;
+        font-family: $font-text;
+    }
+    &__text {
+        font-size: 12px;
+        font-family: $font-text;
+    }
+    &__image-one {
+        width: 90px;
+        height: 90px;
+    }
+    &__image-two {
+        width: 90px;
+        height: 102px;
+    }
+    &__image-three {
+        width: 64px;
+        height: 85px;
+    }
+    &__image-four {
+        width: 75px;
+        height: 74px;
+    }
+}
+@media (min-width: 768px){
+    .cybersecurity {
+        max-width: 1440px;
+        height: 100%;
+        padding: 0 20px;
+        &__section {
+            position: relative;
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            max-width: 1440px;
+            width: 100%;
+            height: 100%;
+            margin: 0 auto;
+        }
+        &__page-title {
+            position: absolute;
```

Comment:
don't need to use position for title here

Reply:
I don’t know why I did it. Done